lib/transports/rest2.js

```diff
@@ -333,6 +334,21 @@ class RESTv2 {
     return this._makePublicRequest(url, cb, Candle)
   }
 
+  /**
+   * Get a list of valid symbol names
+   *
+   * @param {Method?} cb - legacy callback
+   * @return {Promise} p
+   * @deprecated
+   * @see https://docs.bitfinex.com/v2/reference#currency
+   */
+  currency (cb = () => {}) {
+    return this._makePublicRequest(`/conf/pub:map:currency:label`, cb, (data) => {
+      const res = data && data[0]
+      return res.map(row => new Currency(row))
```

Comment:
Make sure to only transform if `this._transform` is true, otherwise return the raw data.